integration/okhttp3/gradle.properties

```diff
@@ -2,14 +2,10 @@ POM_NAME=Glide OkHttp 3.x Integration
 POM_ARTIFACT_ID=okhttp3-integration
 POM_PACKAGING=aar
 
-VERSION_NAME=2.0.0-SNAPSHOT
-VERSION_MAJOR=2
+VERSION_NAME=1.0.0-SNAPSHOT
```

Comment:
I've been leaving these versions the same in all of the integrations just so it's easier to keep track of.